haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="http://www.livejournal.com/~alivejournal/"
+COPYRIGHT="2000 Simon Huet-Sean Graham"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitSrcRevision="11a4bed0a0800f66ce5069d3df6a0a3cc2be43aa"
+SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="198950a11a0cb38cd9093d08619991ad6bbd2d376e7b1999b09a8a556d9add68"
+SOURCE_FILENAME="alivejournal-$portVersion-$gitSrcRevision.tar.gz"
+SOURCE_DIR="alivejournal-$gitSrcRevision"
+PATCHES="alivejournal-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	alivejournal = $portVersion
+	app:AliveJournal = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir $docDir
+	install -t $appsDir objects/AliveJournal
+	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $docDir COPYRIGHT README
+	sed -i -e '/^===$/ d;' $docDir/README
```

Comment:
There's no real need to have a sed command to remove just a "===" line.
I would either drop this sed command or change it a little to get rid of the changelog that is embedded in the README file.

BTW, there is no COPYRIGHT file, so you should remove it from the previous line.

Reply:
Whoops! I'll fix these right away.